devops/providers/azure-devops/templates/azure-pipelines.yml

```diff
@@ -33,7 +33,9 @@ stages:
 - template: ./infrastructure/azure-pipeline-cicd-compose.yml
   parameters:
     environments:
-    - 'devint'
+    - name: 'devint'
+      enablePrIsolation: true
```

Comment:
Should this come from a variable in the var group?

Reply:
Good idea. If it's useful to do that, then we should spin off a post-v1 feature request story for it.